source/iostreams.tex

```diff
@@ -11440,7 +11440,7 @@
 Conformance is specified in terms of behavior. Ideal behavior is not always
 implementable, so the conformance subclauses take that into account.
 
-\rSec3[fs.conform.9945]{POSIX conformance}
+\rSec3[fs.conform.posix]{POSIX conformance}
```

Comment:
Do we really want to change this now that we've published two standards with that stable name?

Reply:
I don't know. Leaving it up to the maintainers.